cfme/tests/webui/test_general_ui.py

```diff
@@ -339,6 +339,34 @@ def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
     assert appliance.advanced_settings["smtp"]["openssl_verify_mode"] == "none"
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(coverage=[1733207])
+@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE)
+def test_vm_right_size_recommendation_back_button(appliance, setup_provider, full_template_vm):
+    """
+    Bugzilla:
+        1733207
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/18h
+        setup:
+            1. Add provider to appliance.
+        testSteps:
+            1. Navigate to a VM's details page.
+            2. From `Configuration` dropdown, select `Right Size Recommendations`.
+            3. Click on `Back` button.
+        expectedResults:
+            1.
+            2.
+            3. Back button must bring back to the details page.
+    """
+    view = navigate_to(full_template_vm, "RightSize")
+    assert view.back_button.is_displayed
```

Comment:
@valaparthvi steps 3 suggesting to `click` on `back_button` and expected result should assert `Details` Page.

Reply:
My bad, I didn't read the testSteps and just wrote the test from what I remembered of the BZ. Made the changes now.